internal/scorecard/plugins/olm_tests.go

```diff
@@ -176,6 +181,42 @@ func NewOLMTestSuite(conf OLMTestConfig) *schelpers.TestSuite {
 
 // Test Implentations
 
+// Run - implements Test interface
+func (t *BundleValidationTest) Run(ctx context.Context) *schelpers.TestResult {
+	res := &schelpers.TestResult{Test: t, MaximumPoints: 1}
+
+	if t.OLMTestConfig.Bundle == "" {
+		res.Errors = append(res.Errors, errors.New("unable to find the OLM 'bundle' directory which is required for this test"))
+		return res
+	}
+
+	// Get the validation API log because it contains
+	// validation output that we include into the scorecard test output.
+	validationLogOutput := new(bytes.Buffer)
+	origOutput := logrus.StandardLogger().Out
+	logrus.SetOutput(validationLogOutput)
+	defer logrus.SetOutput(origOutput)
+
+	_, _, validationResults := manifests.GetManifestsDir(t.OLMTestConfig.Bundle)
```

Comment:
```suggestion
        // The GetManifestDir is an implementation of the API used to this test
        // More info: https://github.com/operator-framework/api/blob/master/pkg/manifests/directory.go
	_, _, validationResults := manifests.GetManifestsDir(t.OLMTestConfig.Bundle)
```

I would add a ref/comment as above for example.

Reply:
It is a nit/suggestion, not a bocker at all.